### ISSUE TITLE
Add custom ping site option 

### ISSUE THREAD

**Issue by be99inner** [dracula/tmux#19]
Condition ping to `github.com` don't work correctly on network.sh.
It's always show Offline. Maybe `github.com` won't response for ICMP.

```bash
if ping -q -c 1 -W 1 github.com &>/dev/null; then
    echo "$(get_ssid)"
else
    echo ' Offline'
fi
```
change dns to fix this.

```bash
ping -q -c 1 -W 1 google.com
```

**Comment by danerwilliams** [comment]
can you give some steps to replicate the issue? I'm able to ping github.com just fine so I think that they respond to ICMP

![Screen Shot 2020-04-28 at 11 18 19 PM](https://user-images.githubusercontent.com/22798229/80565924-8ceadc80-89a6-11ea-9df0-9b3ad667fc11.jpg)

**Comment by be99inner** [comment]
I had tried on my country(Thailand) and some GCP instance on Singapore, github is not response for my ping.
![image](https://user-images.githubusercontent.com/13174138/80566954-4f299a80-8a1e-11ea-8a3c-9675d8976067.png)

**Comment by danerwilliams** [comment]
Ok. I think the best course of action is to create an option for setting a custom site to ping. It seems like there isn’t really a good site that everyone can universally access.

**Comment by danerwilliams** [comment]
Just merged #20 which should fix the issue by using a list of hosts
